Add DCOM Port Range Settings to match Particular Docs

webcentralpa opened the pull request:
[According to the Particular Docs](http://docs.particular.net/nservicebus/operations/transactions-message-processing), the Port Ranges in the "Component
Services" / "Properties for COM Internet Services" dialog should be set
to "5000-6000".

This changeset will update the install-NServiceBusDTC PowerShell cmdlet,
so that it will set these Port Range settings to the recommended values.

This is to resolve issue #28

gbiellem commented:
@webcentralpa  We really appreciate the effort you've gone to here. 

I do have an issue though  - I think the premise that ports 5000-6000 is prescriptive is wrong. Looking at our documentation  I think that's our fault.  We should have said you could use any range that starts above 1024 and the the range should be at least 100 ports.  So I don't believe we should hard code the port range into the cmdlet.  

I think the behavior should be to make the port range an optional parameter.  Without it we don't set the port range. 

If you want to make this change we'd be happy to accept it otherwise I can do the change based on what you've already supplied.

webcentralpa commented:
@gbiellem - thanks for the suggestion. I just added another commit to change the DCOM Port Range setting to an optional named parameter ("PortRange") for the cmdlet . Also, I updated the Install DTC Command's help text in the XML file. (to reflect the changes made in this and my last pull request.)

gbiellem commented:
@webcentralpa  This is great.  Thanks again
